docs_guide/tutorials/tutorials_guidelines.rst

```diff
@@ -0,0 +1,110 @@
+####################
+Tutorials Guidelines
+####################
+
```

Comment:
was this page written in the style of an explanation or how to guide? should it have one of those note boxes at the top?

Reply:
Neither of them. It's just a list of indications/guidelines. We aren't using Diataxis here.